### DISCUSSION TITLE
How does Libretube's "Home" tab choose what set of videos it shows?

### DISCUSSION THREAD

**Discussion by matchboxbananasynergy** [libre-tube/LibreTube#1116] [Q&A] [answered]
I'm assuming that the "Home" tab in Libretube is the equivalent of the "Trending" page in Piped.

- Is it based on IP? (For example, if you're coming from an IP in USA, you'll get USA trending videos, and so on)
- Is it based on the specific instance you've chosen?
- Is it based on something else? (Such as your SIM card's country code or something else)

**Comment by Bnyro** [comment, marked as answer]
It is based on the country you selected in the general settings, by default it's set to auto which means that it uses your phones location/country settings to choose the country.
That process is independent from the IP (which we would never use) and the instance you've chosen.
